help/en/releasenotes/current-draft-note.shtml

```diff
@@ -27,6 +27,8 @@
         <h4>CBUS</h4>
             <ul>
                 <li>Node Manager highlights duplicate CAN IDs.</li>
+                <li>When WiThrottle ecounters an existing non-JMRI Throttle session and is asked Share / Steal / Cancel?,
+                    the main JMRI Throttle silent Steal / silent Share options are checked for preferred behaviour.</li></li>
```

Comment:
`ecounters` is misspelled. `encounters` with the letter `N`.

Reply:
This comment is different from the comment on the help page, help/en/html/hardware/can/cbus/index.shtml .
The two comments might say the same thing but in different ways, but it's confusing.

Reply:
@icklesteve 
Can you check these two review questions?
- `ecounters` is misspelled.
- The comment in `help/en/html/hardware/can/cbus/index.shtml` differ from the comment in `help/en/releasenotes/current-draft-note.shtml`. Is this intended?

Except for these, I think this PR is ready for approval and merge.